src/pro/sky/java/course1/Main.java

```diff
@@ -1,8 +1,112 @@
 package pro.sky.java.course1;
 
+import java.util.Arrays;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        int[] arr = generateRandomArray();
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println(arr[i]);
+        }
+        // work 1
+        int total = 0;
+        for(int i=0;i<arr.length;i++){
+            total = total + arr[i];
```

Comment:
также можно сократить до
```suggestion
            total += arr[i];
```